src/PrettierEditProvider.ts

```diff
@@ -84,7 +84,7 @@ function format(
     let isNonJsParser = false;
     if (!parser) {
         // unset config
-        parser = config.useFlowParser ? 'flow' : 'babylon';
+        parser = 'babylon';
```

Comment:
I'm pretty sure we can't hit this line without having the settings warn. (enum) 
we can remove this block. 
It seems it wasn't working before. :stuck_out_tongue: